### ISSUE TITLE
Error when returning time series of certain frequencies

### ISSUE THREAD

**Issue by nilshg** [JuliaInterop/RCall.jl#376]
```
R> ts(c(1,2,3,4,5,6,7,8,9,10,11,12), start = c(2019, 1), frequency = 12)
ERROR: BoundsError: attempt to access 0-element UnitRange{Int64} at index [1]
```
This seems to only happen when the frequency is 12 and a timeseries has 12+ entries, or the frequency is 4. It looks like it's a display issue, as it's possible to create such a time series in an R string withoug error:
```
R```
test <- ts(c(1,2,3,4,5,6,7,8,9,10,11,12), start = c(2019, 1), frequency = 12)
test[1]
``` 
returns `1`.  So this might be related to R changing the output for quarterly or annual time series?

**Comment by randy3k** [comment]
I cannot reproduce it. What versions of Julia, R and RCall do you have?

<img width="650" alt="Screen Shot 2020-05-04 at 2 13 03 PM" src="https://user-images.githubusercontent.com/1690993/81014256-64813900-8e11-11ea-96d6-c8f25f542310.png">

**Comment by nilshg** [comment]
Hm, weird. Here's a fuller output - ironically calling `version` in R gives the same error. Again this must be in the printing, as I can assign the output of `version` in R and then `@rget` it:

```
julia> versioninfo()
Julia Version 1.4.1
Commit 381693d3df* (2020-04-14 17:20 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, skylake)

(@v1.4) pkg> st RCall
Status `C:\Users\ngudat\.julia\environments\v1.4\Project.toml`
  [6f49c342] RCall v0.13.6

julia> using RCall
[ Info: Precompiling RCall [6f49c342-dc21-5d91-9882-a32aef131414]

R> version
ERROR: BoundsError: attempt to access 0-element UnitRange{Int64} at index [1]

R> a <- version

julia> @rget a
OrderedCollections.OrderedDict{Symbol,Any} with 14 entries:
  :platform         => "x86_64-w64-mingw32"
  :arch             => "x86_64"
  :os               => "mingw32"
  :system           => "x86_64, mingw32"
  :status           => ""
  :major            => "4"
  :minor            => "0.0"
  :year             => "2020"
  :month            => "04"
  :day              => "24"
  Symbol("svn rev") => "78286"
  :language         => "R"
  :version_string   => "R version 4.0.0 (2020-04-24)"
  :nickname         => "Arbor Day"

R> ts(c(1,2,3,4,5,6,7,8,9,10,11,12,13), frequency = 12)
ERROR: BoundsError: attempt to access 0-element UnitRange{Int64} at index [1]
```

**Comment by randy3k** [comment]
I also have julia 1.4.1 and R 4.0.0, but couldn't reproduce it.
I have a macOS if it matters, let me try again with Windows.

**Comment by randy3k** [comment]
I also could reproduce it on windows. Indeed, it is a problem related to printing.

**Comment by randy3k** [comment]
#377 should fix it.
